app.js

```diff
@@ -192,6 +194,17 @@ app.get('/loginFailed', (req, res) => {
     
     };
 
+    // function to show the images from the unsplash API on the imagesGrid page
+    function showImages(req, res){
+      request("https://api.unsplash.com/photos?client_id=RsyiavOTOKJ65Sgr3PECQZ97VYTWbzAwEbHAVoR62Oc", function (error, response, body){
```

Comment:
Be careful with using private keys in the codebase. We can move this the `.env` file